Bonsai.Harp.CF/SyringePumpEvent.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Bonsai.Expressions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reactive.Linq;
+using TResult = System.String;
```

Comment:
This is the using directive from the template and can probably be removed in this case to avoid confusion (I've removed it from the newer Bonsai item templates so for future releases this shouldn't be an issue).